cmd/collection/main.go

```diff
@@ -147,6 +155,14 @@ func main() {
 		// epoch qc contract flags
 		flags.BoolVar(&insecureAccessAPI, "insecure-access-api", false, "required if insecure GRPC connection should be used")
 		flags.StringSliceVar(&accessNodeIDS, "access-node-ids", []string{}, fmt.Sprintf("array of access node IDs sorted in priority order where the first ID in this array will get the first connection attempt and each subsequent ID after serves as a fallback. Minimum length %d. Use '*' for all IDs in protocol state.", common.DefaultAccessNodeIDSMinimum))
+
+		// dynamic node startup flags
+		flags.BoolVar(&dynamicStartup, "dynamic-startup", false, "when set to true the node will attempt to wait for a specified epoch counter & phase before initializing")
+		flags.StringVar(&initANPubkey, "init-access-publickey", "", "the public key of the trusted secure access node to connect to when using dynamic-startup, this access node must be staked")
+		flags.StringVar(&initANAddress, "init-access-address", "", "the access address of the trusted secure access node to connect to when using dynamic-startup, this access node must be staked")
+		flags.StringVar(&startupEpochPhase, "startup-epoch-phase", "EpochPhaseSetup", "the target epoch phase in integer form for dynamic startup <EpochPhaseStaking|EpochPhaseSetup|EpochPhaseCommitted")
+		flags.IntVar(&startupEpoch, "startup-epoch", 0, "the epoch the node will wait for before initializing modules when using dynamic-startup")
```

Comment:
I find it helpful to prefix flag names with the component they are for, especially when there are several related flags.  They show up together in an ordered list of flags (eg. in documentation or CLI help message) and it's easy to see at a glance that a group of flags are configuring the same thing.

```suggestion
		flags.BoolVar(&dynamicStartup, "dynamic-startup", false, "when set to true the node will attempt to wait for a specified epoch counter & phase before initializing")
		flags.StringVar(&initANPubkey, "dynamic-join-access-publickey", "", "the public key of the trusted secure access node to connect to when using dynamic-startup, this access node must be staked")
		flags.StringVar(&initANAddress, "dynamic-join-access-address", "", "the access address of the trusted secure access node to connect to when using dynamic-startup, this access node must be staked")
		flags.StringVar(&startupEpochPhase, "dynamic-join-epoch-phase", "EpochPhaseSetup", "the target epoch phase in integer form for dynamic startup <EpochPhaseStaking|EpochPhaseSetup|EpochPhaseCommitted")
		flags.IntVar(&startupEpoch, "dynamic-join-epoch-counter", 0, "the epoch the node will wait for before initializing modules when using dynamic-startup")
```

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/8ea656ac507e9969c635a600e908891394d7ac2c